docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -137,21 +93,14 @@ To start a validator, run the following commands from a Linux terminal:
 .. code-block:: console
 
    $ sawtooth keygen --key-dir /etc/sawtooth/keys/ validator
-   $ validator -vv --endpoint tcp://localhost:8800
+   $ validator -vv --endpoint tcp://127.0.0.1:8800 \
```

Comment:
--endpoint is not necessary in this context as it default to the network bind string.